lib/models/user.rb

```diff
@@ -0,0 +1,28 @@
+require 'bcrypt'
+
+class User
+#
+  include DataMapper::Resource
+
+  attr_reader :password
+  attr_accessor :password_confirmation
+  validates_confirmation_of :password
+
+  property :id, Serial
+  property :email, String, required: true, unique: true
+  property :username, String
+  property :password_digest, Text
+
+  has n, :peeps, through: Resource
+
+
+  def password=(password)
+    @password = password
+    self.password_digest = BCrypt::Password.create(password)
+  end
+
+  def self.authenticate(email, password)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `password`. If it's necessary, use `_` or `_password` as an argument name to indicate that it won't be used.